packages/spotlight/src/container.js

```diff
@@ -89,7 +90,10 @@ let GlobalConfig = {
  * @private
  */
 const querySelector = (node, includeSelector, excludeSelector) => {
-	const include = new Set(node.querySelectorAll(includeSelector));
+	let include = new Set(node.querySelectorAll(includeSelector));
```

Comment:
Let's move `querySelectorAll` call into two lines below as we a new `Set` here is not used actually.

Reply:
It's fixed.